.github/workflows/rust.yml

```diff
@@ -64,8 +64,11 @@ jobs:
     - run: cargo clippy --all-targets --target ${{ matrix.target }}
     - run: make build-test-wasms
     - if: startsWith(matrix.target, 'x86_64')
-      run: cargo test --workspace --target ${{ matrix.target }}
-
+      # specify directories explicitly to avoid building the preflight library (otherwise it will fail with missing symbols)
+      run: |
+        for I in cmd/soroban-cli cmd/soroban-cli/tests/fixtures/test-wasms/hello_world cmd/soroban-cli/tests/fixtures/test-wasms/invoker_account_exists; do
+          cargo test --target ${{ matrix.target }} $I
+        done
```

Comment:
You can prevent the preflight crate from being tested by adding the following to its Cargo.toml:
```
[lib]
test = false
doctest = false
```
I think if you add that, you probably don't need to do this loop.

Reply:
I may end up adding tests to the library. The loop is there because trying to run the tests is only problematic in some architectures (e.g. windows, if I recall properly)